js_modules/dagit/packages/core/src/testing/defaultMocks.ts

```diff
@@ -14,6 +14,11 @@ export const defaultMocks = {
   Asset: () => ({
     id: randomId,
   }),
+  AssetNode: () => ({
```

Comment:
This separately fixed another failure I saw locally which oddly Buildkite isn't complaining about.